Assignment2/src/BoardFrame.java

```diff
@@ -333,7 +323,7 @@ public void mouseReleased(MouseEvent e) {
 		int x = e.getX();
 		int y = e.getY();
 		
-		Position pos = new Position(x/SQUARE_WIDTH,y/SQUARE_HEIGHT);
+		Position pos = new Position(x/36,y/36);
```

Comment:
There are constants for these? or do they need to be updated, or are these different values for a reason

Reply:
Please ignore it. It should be as before.